src/components/movie-card/movie-card.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC } from "react";
+import styles from "./movie-card.module.scss";
+
+interface MovieCardProps {
+  img: string;
+  title: string;
+  year: number;
+  description: string;
+}
+export const MovieCard: FC<MovieCardProps> = ({
+  img,
+  title,
+  year,
+  description,
+}) => {
+  return (
+    <div className={styles.card}>
+      <img src={img} alt="" className={styles.img} />
```

Comment:
Well adding `alt` text is a good case especially when users have slow Internet or link URL has been changed. Let's also pass props with this text, but make it optional and use `{title}-poster` value if alt was not provided